package j20_예외처리;

public class ExceptionTest1 {
	public static void main(String[] args) {
		int[] numArray = {1,2,3,4,5};
		
		for(int i=0; i<numArray.length;i++)
		{
			try {
				System.out.println(numArray[i]);
			} catch (Exception e) {
				System.out.println("배열 갯수 초과");
			}
			
		
		
		}
		System.out.println("반복 후");
		
		
		
		}

}
